LongWarOfTheChosen/Config/XComGameData.ini

```
@@ -1673,6 +1673,8 @@ ActionPriority=30
 MaxPerMonth=3
 GlobalCooldown=0
 MinXComKnowledge=4
+-ActivationTime=(MinDays=4, MaxDays=20)
++ActivationTime=(MinDays=2, MaxDays=10)
```

This isn't an array, so I'm not sure what this syntax will do. I would just stick to:
```suggestion
ActivationTime=(MinDays=2, MaxDays=10)
```